includes/wikia/parser/templatetypes/tests/TemplateTypesParserTest.php

```diff
@@ -27,7 +27,7 @@ public function testTemplateParsing( $config, $templateText, $expectedValue, $me
 
 		TemplateTypesParser::onFetchTemplateAndTitle( $templateText, new Title );
 
-		$this->assertEquals( $expectedValue, $templateText, $message );
+		$this->assertEquals( $expectedValue, $templateText );
 	}
 
 	public function TemplateParsingDataProvider() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This function "TemplateParsingDataProvider" has 298 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)